requirements.txt

```diff
@@ -0,0 +1,83 @@
+absl-py==0.13.0
```

Comment:
Can this requirements list be compacted? I realize getting accurate library versions is important for reproducibility, but it is also spooky and may also break down because of fixed version numbers (e.g. Windows does not have prebuilt binaries, or M1 system requires a newer version).

Will just leaving in the minimal ones (e.g. stable-baselines, minerl, wandb, etc) be enough?

Reply:
That's a good point. Will do.